datalab/context/_utils.py

```diff
@@ -75,9 +79,10 @@ def get_credentials():
       # Use the first gcloud one we find
       for entry in creds['data']:
         if entry['key']['type'] == 'google-cloud-sdk':
-          return oauth2client.client.OAuth2Credentials.from_json(json.dumps(entry['credential']))
+          creds = oauth2client.client.OAuth2Credentials.from_json(json.dumps(entry['credential']))
+          return google.auth._oauth2client.convert(creds)
```

Comment:
Might be worthwhile to just create a new credential out of the old one using the impl (https://github.com/GoogleCloudPlatform/google-auth-library-python/blob/master/google/auth/_oauth2client.py#L58) of the converter, in case they rename/move the file since it is not a public API yet. I guess it's Okay to use the internal converter too since we pin the version. Up to you.

Reply:
Fair point. Done.